index.html

```diff
@@ -5129,7 +5131,7 @@ <h2>
         <p>
           To help ensure that users do not inadvertently share sensitive
           credentials with an origin, this API requires that PaymentRequest's
-          <a>show()</a> method be <a>triggered by user activation</a> (e.g.,
+          <a>show()</a> method be triggered by [=transient activation=] (e.g.,
```

Comment:
To fully eliminate old model's token passing idea, I suggest dropping "triggered by" completely.  What about this: "... `show()` method be invoked while the relevant `Window` has transient activation"?